python/lsst/afw/math/interpolate.cc

```diff
@@ -71,25 +71,16 @@ PYBIND11_PLUGIN(_interpolate) {
                        (ndarray::Array<double, 1> (Interpolate::*) (ndarray::Array<double const, 1> const&)
                            const) &Interpolate::interpolate);
 
-    mod.def("makeInterpolate", 
-                       (PTR(Interpolate) (*)(std::vector<double> const &,
-                                             std::vector<double> const &,
-                                             Interpolate::Style const)) makeInterpolate,
-                       "x"_a, "y"_a, "style"_a=Interpolate::AKIMA_SPLINE);
-    mod.def("makeInterpolate", 
-                       (PTR(Interpolate) (*)(ndarray::Array<double const, 1> const &,
-                                             ndarray::Array<double const, 1> const &y,
-                                             Interpolate::Style const)) makeInterpolate,
-                       "x"_a, "y"_a, "style"_a=Interpolate::AKIMA_SPLINE);
-    /* Module level */
+    mod.def("makeInterpolate",
+            (std::shared_ptr<Interpolate>(*)(std::vector<double> const &, std::vector<double> const &,
```

Comment:
I think you mean adding a space before "makeInterpolate"? I agree it looks funny without, but unfortunately clang-format did it this way, so I'm inclined to live with it. (@pschella agreed in private conversation)